util/util.go

```diff
@@ -213,8 +213,8 @@ func PortionAfterSep(input, sep string, occ int) string {
 // Right now, this is used only to decide whether to print the location of the producer expression
 // in a full trigger.
 func ExprIsAuthentic(pass *analysis.Pass, expr ast.Expr) bool {
-	_, ok := pass.TypesInfo.Types[expr]
-	return ok
+	t := pass.TypesInfo.TypeOf(expr)
+	return t != nil
```

Comment:
This one is also an unrelated fix, right? Looks good to me, just confirming it hasn't much to do with the PR goal.

Reply:
No, this is actually related a fix. The original [TypesInfo.Types](https://pkg.go.dev/go/types#Info.Types) does not capture all identifiers, such as `var i *int`. While, [TypesInfo.TypeOf](https://pkg.go.dev/go/types#Info.TypeOf) comprehensively captures all expressions.

So, in the simple code example below, position for uninitialized variable `i` was not being reported earlier, which works fine now.

```
var i *int
print(*i)
```